src/OpenSpaceToolkit/Physics/Time/Instant.cpp

```diff
@@ -374,14 +374,20 @@ Int64                           Instant::getLeapSecondCount                 ( )
     return Instant::dAT_UTC(this->inScale(Scale::UTC).count_) / 1000000000 ;
 }
 
-String                          Instant::toString                           (   const   Scale&                      aTimeScale                                  ) const
+String                          Instant::toString                           (   const   Scale&                      aTimeScale,
+                                                                                const   DateTime::Format&           aDateTimeFormat                             ) const
 {
 
     if (!this->isDefined())
     {
         throw ostk::core::error::runtime::Undefined("Instant") ;
     }
 
+    if ((aTimeScale == Scale::UTC) && (aDateTimeFormat == DateTime::Format::ISO8601))
+    {
+        return this->getDateTime(aTimeScale).toString(aDateTimeFormat) + "Z" ;
+    }
+
     return this->getDateTime(aTimeScale).toString() + " [" + StringFromScale(aTimeScale) + "]" ;
```

Comment:
What about something like:

```suggestion
    if ((aDateTimeFormat != DateTime::Format::Standard) && (aTimeScale == Scale::UTC))
    {
        return this->getDateTime(aTimeScale).toString(aDateTimeFormat) + "Z" ;
    }
    
    return this->getDateTime(aTimeScale).toString(aDateTimeFormat) + " [" + StringFromScale(aTimeScale) + "]" ;
```

This way the newly added `aDateTimeFormat` arg can be used in all cases.

Reply:
Or
```suggestion
    if ((aDateTimeFormat == DateTime::Format::ISO8601) && (aTimeScale == Scale::UTC))
    {
        return this->getDateTime(aTimeScale).toString(aDateTimeFormat) + "Z" ;
    }
    
    return this->getDateTime(aTimeScale).toString(aDateTimeFormat) + " [" + StringFromScale(aTimeScale) + "]" ;
```
since the `Z` suffix is ISO 8601 specific? No?

Reply:
Yeah that's correct. I believe this works well and addresses Lucas' point.